benchmarks/NonlinearProblem/nonlinear_solver_23_tests.jmd

```diff
@@ -565,7 +565,6 @@ end
 save("summary_wp_23test_problems.svg", fig)
 ```
 
-# Appendix
```

Comment:
Revert this? I don't understand why we would remove the standard appendix header

Reply:
This one is duplicated, see:
https://github.com/SciML/SciMLBenchmarks.jl/blob/edcbe452deb980f9379204344bc6e525067cd846/src/SciMLBenchmarks.jl#L89-L96

And

https://docs.sciml.ai/SciMLBenchmarksOutput/dev/NonlinearProblem/nonlinear_solver_23_tests/#Appendix

Reply:
gotcha